diff --git a/node_modules/dts-bundle-generator/dist/types-usage-evaluator.js b/node_modules/dts-bundle-generator/dist/types-usage-evaluator.js
index a09f141..9f25078 100644
--- a/node_modules/dts-bundle-generator/dist/types-usage-evaluator.js
+++ b/node_modules/dts-bundle-generator/dist/types-usage-evaluator.js
@@ -111,6 +111,9 @@ class TypesUsageEvaluator {
         if (ts.isExportDeclaration(node) && node.exportClause !== undefined && ts.isNamedExports(node.exportClause)) {
             for (const exportElement of node.exportClause.elements) {
                 const exportElementSymbol = (0, typescript_1.getImportExportReferencedSymbol)(exportElement, this.typeChecker);
+                if(exportElementSymbol === undefined) {
+                    continue;
+                }
                 // i.e. `import * as NS from './local-module'`
                 const namespaceImportForElement = (0, typescript_1.getDeclarationsForSymbol)(exportElementSymbol).find(ts.isNamespaceImport);
                 if (namespaceImportForElement !== undefined) {
